### PR TITLE
feat: add ASCII

### PR DESCRIPTION
**Pull request by marktoda**
## Related Issue
Which issue does this pull request resolve?

## Description of changes

### CODE REVIEW

File: src/PoolManager.sol [hunk 1]
```diff
@@ -26,6 +26,54 @@ import {Extsload} from "./Extsload.sol";
...
+//        4444            4444     4444
+//          d444          4444444    4444                           4
+//            4444        44444444     4444                         4
+//             44444       4444444       4444444444444444       444444
+//           4   44444     44444444       444444444444444444444    4444
+//            4    44444    4444444         4444444444444444444444  44444
+//             4     444444  4444444         44444444444444444444444 44  4
+//              44     44444   444444          444444444444444444444 4     4
+//               44      44444   44444           4444444444444444444 4 44
+//                44       4444     44             44444444444d444     444
```
Comment (hensha256) [inline src/PoolManager.sol:43]:
```suggestion
//                44       4444     44             444444444444444     444
```

Comment (hensha256) [reply src/PoolManager.sol:43]: theres a bunch of `d`s are they purposeful?

Comment (marktoda) [reply src/PoolManager.sol:43]: oo no i can s/d/4

File: src/PoolManager.sol [hunk 2]
```diff
@@ -26,6 +26,54 @@ import {Extsload} from "./Extsload.sol";
 
 /// @notice Holds the state for all pools
```
Comment (hensha256) [inline src/PoolManager.sol:27]: I feel like this line should go directly above
```
contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claims, Extsload {
```

? thoughts?

Comment (marktoda) [reply src/PoolManager.sol]: agreed fixed

**Review by snreynolds — APPROVED** [review]

**Pull request merged**